### ISSUE TITLE
fetching rrd information

### ISSUE THREAD

**Issue by oetiker** [oetiker/rrdtool-1.x#204]
Hello,

I am really new to the RRD tool. I need your assistance in fetching the 1 month data from the RRD.

for example I am issuing the below command but it seems not logically correct number of rows being fetched from the rrd/rra.

rrdtool fetch gauge_swp_free_pct.rrd -r 300 AVERAGE -s -30d

this fetched me total records of 1443 but actually 30days of records for every 5 minutes data capture should come to closer 8931 or so.

if I rrdtool dump <rrd file name>, I can see data for so many previous months.

I am not sure where I am going wrong. Is it expected behaviour ?

In summary I need to capture the 1 month data (which was captured for every 5 minutes) from the RRD.

Please assist at the earliest.

Thank you,
Regards,
Ravi
